test/unit/nodes/element.js

```diff
@@ -18,6 +18,23 @@ suite('Element', function() {
 			);
 		});
 
+		suite('#querySelectorAll', function() {
+			test('css selector with tag name only', function() {
+				var lis = elem.querySelectorAll('li');
```

Comment:
To protect against naive implementations (such as `querySelectorAll = getElementsByTagName`, could you overspecify this selector? Something like `div.a ul#unordered.b.c li` would do the trick